tools/performance/GC.wprp

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WindowsPerformanceRecorder Version="1.0">
+  <Profiles>
+    <EventCollector Id="EventCollector_MyEventSource" Name=".NETSession">
+      <BufferSize Value="1024" />
+      <Buffers Value="1024" />
```

Comment:
That's... 1 GB of buffers? Seems a bit excessive.

Reply:
Indeed. I forget where I copied that profile from and it looks like it's not actually being used so I'll just delete the file.